README.md

```diff
@@ -36,46 +36,61 @@ Finally the user can use the application in three ways
  3. In the side panel of the file application, a 'Persons' tab is added where
     you can see a list of your friends in the photo, and rename them.
 
-## Installation
+## Installation, configuration and usage
 
 #### Requirements
 
- * Nextcloud 14+
+ * Nextcloud 16+
  * [Dlib PHP bindings](https://github.com/goodspb/pdlib)
+ * 2GB of RAM
```

Comment:
2GB is not hard requirement anymore, not sure why you are adding this? Maybe add it, but with some disclaimer:)

Reply:
In the last releases was established 2 GB of ram as a general requirement, with the excuse that the clustering of many photos can consume a lot of memory.

Then it requires 1GB as "memory_limits" to PHP for cnn models.